### ISSUE TITLE
How about considering looking for maintainers?

### ISSUE THREAD

**Issue by dotneet** [mckaywrigley/chatbot-ui#682]
Hi @mckaywrigley,

First of all, I would like to thank you for developing a great OSS.

It seems like there are too many pull requests and issues for one person to maintain this project.
So, how about adding more maintainers?

**Comment by jmagly** [comment]
seconded, im pretty sure things im working on can be contributed back, for example the other issue I have posted deals with changes OpenAI made this weekend. I have a dirty fix but am thinking of pushing a better one given I can do it in line with the project goals and I know it would be accepted if I spent the time/effort.

**Comment by Xav-Pe** [comment]
Yeah is this project abandonned?

**Comment by jmagly** [comment]
there are updates from time to time but the creator has a bunch of projects. i can push my system changes to my fork and folks can use it until something updates here.

**Comment by reconrad48** [comment]
jmagly, what updates are you working on for chatbot ui?

**Comment by ufulu** [comment]
@jiayao I would greatly apreciate that, thanks!

**Comment by Maxiviper117** [comment]
This repo is slowly getting out of hand and will soon become another abandoned project . Just having simple things like github issue labels to have some organization and enabling the discussion for more open talk.

For this project to continue, there needs to be a community setup. This Mckay fella has a developer ADD, any shiny new thing he sees he dumps a lot time into that, as we can see on his Twitter, he got no time for this repo anymore.

**Comment by jmagly** [comment]
@reconrad48 right now I have just posted my fix to force all messages through the system role. The point of this is to restore the functionality we had last week before the user role was nerfed in this weekend's update. 

You'll see I've got the "fix" in a PR here but its mostly just a patch as im sure some people will want to use the roles in the way the software is setup. 

im not sure what the ideal UX solution is here but I want to be able to pick any role the API accept and inject any data i like into it. This may be more like a dev tool and not useful here so I would be curious to know what balance should be struck among other users. 

if this repo is being abandoned I can start homing out of my repo and can accept PRs there but I might need at least 1 or 2 other people both qualified and willing to help approve things and move stuff forward. 

I intend on using this project as a start point for some products I am working on so having help maintaining a simple and clean "baseline" app would be very ideal for me at least.

**Comment by reconrad48** [comment]
Exactly, I am glad someone other than me is paying attention. Dude created this and then immediately went silent, unreachable... I am still thankful that he put together working code I would become overwhelmed with the amount of issue requests. However, due to another github project getting hacked, I no longer trust anything by default. The lenc chatgpt desktop app made over a hundred dollars API calls from my account and even after warning everyone on it, people are still using it 3 weeks after it was compromised... people haven't noticed that the original creator has abandoned the project 5 weeks ago... and yet comment after comment about bugs are still pouring in..  I think this project will be the same. Although I am looking through the code to see if there are backdoors etc. It would be nice if there were github watchdogs that are very experienced in various programming languages that would monitor more popular projects and place a "safe badge" on them... other than that, github, with the popularly of GPT, can quickly becoming a breeding ground for hackers.

**Comment by reconrad48** [comment]
@jmagly I went to openAI to look for this update you have mentioned, and I haven't found anything since early march... announcing the plugin beta wait list etc. Nothing in my inbox either.. do you have a link to the update?

**Comment by reconrad48** [comment]
I think every repo project should have at least one credible person to sign off for security issues.

**Comment by jmagly** [comment]
@reconrad48  I've not found any announcements on any of the updates that have been made to the system but they make them pretty much weekly. I'm not poking or toying, Im actively building an application with processes that are validated multiple times (some i do daily). When things change, its noticeable, in part due to the nature of my application. Most of my development prompts only work in 4, they will crash 3.5. 

However this weekend's change has been the biggest change Ive seen yet. My once working development prompts suddenly and for the first time since the system went online, began consistently failing in a hard halt as if I was running 3.5. After some testing between builds and the usual troubleshooting I discovered the system role, which has been known to be weak and not able to prevent jailbreaks had been significantly buffed while the user role had been weakened. Makes for easier building of apps that can't be jailbroken but certainly put a cramp on my dev process for a day. 

fwiw, my background is in industrial safety and secure data processing for banks and fintech.

**Comment by dotneet** [comment]
I received a request on Twitter to set up a Discord server as a community for chatbot-ui.
It may take some time for Mckay to become aware of this Issue, so I will try to set up a Discord server as a temporary community.

Please let me know if there are any problems with this or if there is a better way to do this.

**Comment by jmagly** [comment]
mckay seems like hes busy riding the AI wave influencer dev style while building something. I suspect the toy gets ignored until there is time unless its part of daily workstream. lets do this i guess, if anything some one will scream once we start organizing, post the link

**Comment by dotneet** [comment]
I've created the Discord server.
https://discord.gg/VkTfdnQR

**Comment by rdvo** [comment]
anyone mainaining this?  Discord link is expired

**Comment by thebalaa** [comment]
Sad to see this project become abandonware with so many people showing up willing to contribute. Let's compile a list of alternatives with active maintainers and move our efforts there.

**Comment by rdvo** [comment]
yeah is there a  discord thats active? Let get a fork up omewhere with active maintainers and keep it moving.. lot of people want an open source alternative to chatgpt interface but still powered by the api and potnetially plugins etc

thanks!

**Comment by rossman22590** [comment]
I have been maintaining it on my end, is there a need for someone to take charge?

**Comment by itbm** [comment]
I believe there is a fork by @jorge-menjivar with discord server: https://github.com/jorge-menjivar/chatbot-ui

**Comment by fox4snce** [comment]
This project is dead unless people can agree on a single fork to work on and migrate to.  Jorge's last update was 2 weeks ago.  That's no much better than mckays.

**Comment by jorge-menjivar** [comment]
Been working hard on a major changes. Don't want to push something unfinished.

**Comment by rossman22590** [comment]
Jorge I’m happy to work with you I have a team that can help us (my team)

**Comment by itbm** [comment]
@jorge-menjivar could you enable issues on your fork? Not everyone will have Discord.

**Comment by jorge-menjivar** [comment]
@rossman22590 Sounds good. Let me know on social or discord how we can collaborate together.

**Comment by jorge-menjivar** [comment]
@itbm Done. I also enabled discussions.

**Comment by jorge-menjivar** [comment]
Btw my latest work is being published to the 'experimental' branch.

**Comment by rossman22590** [comment]
@jorge-menjivar DM me @tsi_org on twitter https://twitter.com/tsi_org . We have a nitro discord too we can use!

**Comment by khalidjaz** [comment]
I'm on the run now - but I have it running on my k8s cluster and have paying users, I have been maintaining it quite a bit, if there might be a join of forces on this I'm happy to help.

**Comment by rdvo** [comment]
i ave no problem paying or supporting maintainers of this,  we also have gpt32K access if anyone wants to pair code and build out some stuff.. reach out to me rob[at]consoles.ai

**Comment by jorge-menjivar** [comment]
I have some very exciting news that I would love to share with everyone. Super exciting partnerships to continue development of the project, new features and access to lots of secret stuff 🤫. All this will make what we have right now look insignificant. Information about this will be released early next week. Stay tuned and hit me up on social if you are interested in partnering up.

**Comment by rdvo** [comment]
> I have some very exciting news that I would love to share with everyone. Super exciting partnerships to continue development of the project, new features and access to lots of secret stuff 🤫. All this will make what we have right now look insignificant. Information about this will be released early next week. Stay tuned and hit me up on social if you are interested in partnering up.

will you be maintaining a new repo?

**Comment by jorge-menjivar** [comment]
Yes, in conjunction with a group of other people.

**Comment by thesofakillers** [comment]
> Information about this will be released early next week

@jorge-menjivar has this information been released? Where?

**Comment by luiztauffer** [comment]
any updates on this? It's really a pity seeing so much community engagement wasted due to abandonment

**Comment by jorge-menjivar** [comment]
Sorry for the long wait everybody. I was debating on the steps to take. I decided to go a different route and release my fork as a separate project since the code has is too different to merge upstream now. I released as much work as I could while taking into consideration the installation complexity. By default, my project now runs on Supabase only to have a straightforward, yet capable solution. The name of the new project is called unSAGED, I hope you like it!

![Screenshot from 2023-07-01 03-11-06](https://github.com/mckaywrigley/chatbot-ui/assets/16660534/4b843663-9d6b-4937-b7e1-2652bc8049a5)
